package.json

```diff
@@ -40,7 +40,7 @@
     "hellojs": "1.14.1",
     "local-storage": "1.4.2",
     "lodash": "4.17.4",
-    "loglevel": "1.4.1",
+    "loglevel": "github:thomasconner/loglevel",
```

Comment:
I submitted a [PR](https://github.com/pimterry/loglevel/pull/104) that was merged into the loglevel master branch. We can use the latest release of loglevel instead of a personally maintained version.